src/main/webapp/content/scss/partials/_d3-visualizations.scss

```diff
@@ -347,4 +347,42 @@ line {
 .dataTables_scrollBody::-webkit-scrollbar-thumb {
     background-color: rgba(136, 139, 142, 0.5);
     border-radius: 2px; 
-}
\ No newline at end of file
+}
```

Comment:
I think the tooltip css is already present in the flair-visualizations repo which gets bundled in the main.bundle.js file. Why has this been kept in this file separately again?